docs/source/cn/activation.py

```diff
@@ -138,3 +138,36 @@
 
     """,
 )
+reset_docstr(
+    oneflow.nn.functional.conv1d,
+    r"""
+    conv1d(input, weight, bias=None, stride=[1], padding=[0], dilation=[1], groups=1) -> Tensor
+
+    文档引用自： https://pytorch.org/docs/stable/generated/torch.nn.functional.conv1d.html?highlight=conv1d
+
+    对由多个输入平面组成的输入信号应用一维卷积。
+
+    请参阅：class:`~oneflow.nn.Conv1d` 获取有关详细信息和输出形状。
+
+    参数：
+        input: 形状的量化输入张量：math:`(\text{minibatch} , \text{in_channels} , iW)`
```

Comment:
```suggestion
        input: 形状的量化输入张量 :math:`(\text{minibatch} , \text{in_channels} , iW)`
```
前后的冒号都应该是英文才能识别，并且前后都应该空格

Reply:
后面的空格指在第二个 ` 后面空格